# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1443260539 -7200
#      Sat Sep 26 11:42:19 2015 +0200
# Branch stable
# Node ID 63d14a5700396edc3da1b6437196a4a86e56666a
# Parent  0fdba3fdf40ea1dc28319a4a38d10289ea4aaa2f
annotation.m: document the "string" property of textbox and textarrow annotations (bug #4603.6)

* annotation.m: add a doc string about "string" property of textbox and textarrow annotations

diff --git a/scripts/plot/appearance/annotation.m b/scripts/plot/appearance/annotation.m
--- a/scripts/plot/appearance/annotation.m
+++ b/scripts/plot/appearance/annotation.m
@@ -95,28 +95,30 @@
 ## @qcode{"head1length"}/@qcode{"head2length"},
 ## @qcode{"head1width"}/@qcode{"head2width"}, etc.  The index 1 marks the
 ## properties of the arrowhead at the first point in @var{x} and @var{y}
 ## coordinates.
 ##
 ## @item @qcode{"textarrow"}
 ## Construct an arrow with a text label at the opposite end from the arrowhead.
 ##
+## Use the "string" property to change the text string.
 ## The line and the arrowhead can be customized as for arrow annotations, and
 ## the text can be customized using the same properties as @code{text} graphics
 ## objects.  Note, however, that some text property names are prefixed with
 ## "text" to distinguish them from arrow properties:
 ## @qcode{"textbackgroundcolor"}, @qcode{"textcolor"},
 ## @qcode{"textedgecolor"}, @qcode{"textlinewidth"},
 ## @qcode{"textmargin"}, @qcode{"textrotation"}.
 ##
 ## @item @qcode{"textbox"}
 ## Construct a box with text inside.  @var{pos} specifies the
 ## @qcode{"position"} property of the annotation.
 ##
+## Use the "string" property to change the text string.
 ## You may use @qcode{"backgroundcolor"}, @qcode{"edgecolor"},
 ## @qcode{"linestyle"}, and @qcode{"linewidth"} properties to customize
 ## the box background color and edge appearance.  A limited set of @code{text}
 ## objects properties are also available; Besides @qcode{"font@dots{}"}
 ## properties, you may also use @qcode{"horizontalalignment"} and
 ## @qcode{"verticalalignment"} to position the text inside the box.
 ##
 ## Finally, the @qcode{"fitboxtotext"} property controls the actual extent of
